config/owasp/suppressions.xml

```diff
@@ -1,14 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?><suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+
+  <!--Please add all the false positives under the below section-->
+  <suppress>
+    <notes>False Positives
+      CVE-2016-3094 refer https://tools.hmcts.net/jira/browse/CCD-4056
+    </notes>
+    <cve>CVE-2016-3094</cve>
+  </suppress>
+  <!--End of false positives section -->
+
+  <!--Please add all the temporary suppressions under the below section-->
   <suppress>
     <notes>Temporary Suppression
-        CVE-2022-41946 refer [Ticket]
-        CVE-2016-4432 refer https://tools.hmcts.net/jira/browse/CCD-4056
-        CVE-2016-3094 refer https://tools.hmcts.net/jira/browse/CCD-4056
-        CVE-2016-4432 refer [Ticket]
-        CVE-2016-3094 refer [Ticket]
-        CVE-2021-37533 refer [Ticket]</notes>
+      CVE-2022-41946 refer [Ticket]
```

Comment:
@JamesCollettCGI - add the ticket details for these CVEs.

Reply:
Fixed CVE-2022-41946 (postgresql version)   and   Added CVE-2021-37533 ticket ref.

Reply:
CVE-2022-41946 is already fixed and ready to be deployment.

Reply:
@JamesCollettCGI - can you resolve the conflicts?